docs/package-requirements.md

```diff
@@ -0,0 +1,22 @@
+# Package Requirements
+
+A set of criteria used to evaluate packages used by [GitHub](https://github.com).
+
+* MUST be published on [npm registery](https://www.npmjs.com).
+* SHOULD be also available on [GitHub](https://github.com) with tagged releases.
+* MUST provide an [OSI](https://opensource.org/licenses) license file and declare in `package.json` `"license"` field.
+* SHOULD be available under an MIT license.
+* MUST follow [SemVer](http://semver.org) versioning.
```

Comment:
Yeah, maybe I can just remove this line. *Most* packages follow some kind of reasonable versioning policy.